chat-commands.js

```diff
@@ -2921,7 +2921,9 @@ exports.commands = {
 		if (!user.hasConsoleAccess(connection)) {
 			return this.errorReply("/bash - Access denied.");
 		}
-
+		if (!target) {
+			return this.sendReply("/bash (Console Access) needs arguments to carry out execution by running a child process. \nExample Execution: /bash ls");
```

Comment:
No, you're not stupid, it just wasn't your first instinct; nothing is wrong with that!